### PR TITLE
remove unused/old pages from site

### PR DESCRIPTION
**Pull request by sqs**
Removes unused/old pages from the site to reduce the surface area of the site, which will make it easier to keep up to date with our latest messaging and dev-focused approach. I decided to make this kind of cleanup when I realized that there were so many different variants of pages, and many that had old messaging or were not targeted at devs (which would feel out of place now).

- remove "universal code search ebook"
- remove /partner-podcast page: No pages linked to this, and there was no traffic to it.
- remove most press releases (move big code survey 2020 to blog)
- remove old "remote work made easier" white paper: It got <6 visits per week and is outdated.
- remove /news page with articles to press mentions: This was not updated, and people can just Google for mentions.
- clean up navlinks: sentence case, remove link to dev tool time (not updated in 1 year)
- remove "dev tools quiz"

### CODE REVIEW

File: src/components/Layout/navLinks.tsx [hunk 1]
```diff
@@ -142,12 +142,12 @@ export const footerLinks: NavLink[] = [
                 href: '/blog',
             },
             {
-                title: 'Changelog',
-                href: 'https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/CHANGELOG.md',
+                title: 'Docs',
```
Comment (bretthayes) [inline src/components/Layout/navLinks.tsx:145]: Do we want to keep the duplicate Docs nav item that's outside of this Resources dropdown?

**Review by bretthayes — APPROVED** [review]
I just have some comments, but otherwise looks good!

File: src/components/Layout/navLinks.tsx (hunk 1 above)
Comment (sqs) [reply src/components/Layout/navLinks.tsx:145]: Nope, thanks. Will fix.

File: netlify.toml [hunk 2]
```diff
@@ -155,6 +155,11 @@
   from = "/press-release/*"
   to = "/press/:splat"
 
+[[redirects]]
+  from = "/press/big-code-survey-2020"
+  to = "/blog/big-code-survey-2020"
+
+
```
[the remark this answers is not in the document]
Comment (sqs) [reply netlify.toml:162]: No...I don't think it's worth it.

**Comment by sqs** [comment]
@sourcegraph/stampers Already reviewed and approved but then made a minor add'l edit. A stamp would be appreciated!

**Review by BolajiOlajide — APPROVED** [review]

**Pull request merged**